app/views/candidate_interface/contact_details/address/_form_fields.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= f.govuk_error_summary %>
+
+<% if @contact_details_form.uk? %>
+  <fieldset class="govuk-fieldset">
```

Comment:
Can we use `govuk_fieldset` from the form builder for this?

Reply:
@paulrobertlloyd - Sorry not sure I follow, The fieldset's class is already set to "govuk-fieldset"? does this need to be an underscore?

Reply:
Sorry, should have been clearer. I meant the `govuk_frontend` helper from the form builder gem. See: https://govuk-form-builder.netlify.app/building-blocks/fieldsets/